SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs

```diff
@@ -5,10 +5,11 @@
 
 namespace SeaPublicWebsite.Models.EnergyEfficiency
 {
-    public class YourSavedRecommendationsViewModel
+    public class ActionPlanViewModel
     {
         public PropertyData PropertyData { get; set; }
         public string EmailAddress { get; set; }
+        public string BackLink { get; set; }
```

Comment:
You could inherit `QuestionFlowViewModel` instead of adding a backlink field

Reply:
But it's not part of the question flow :)

Reply:
True...